week-3/B-array-some/exercise.js

```diff
@@ -21,4 +21,8 @@ var pairs = pairsByIndex.map(function(indexes) {
   return [student, mentor];
 });
 
-console.log(pairs);
```

Comment:
You've changed this line :slightly_smiling_face: . Re-read the instructions at the top of the file and make sure the program does what it's expected to do when you run it. I think there was a previous example on how to exit a program, but you can search for it too.